tests.md

```diff
@@ -0,0 +1,239 @@
+# Tests
+
+The WordPress Hosting Team provides tools for hosting companies to run the WordPress automated tests on their infrastructure to improve compatibility with WordPress. These results can be published them on the [Host Test Result information page](https://make.wordpress.org/hosting/test-results/), to help WordPress' compatibility with hosts as well.
+
+It consists of two tools: the Runner, which is part of the core's PHPUnit tests, which work on a hosting, and optionally [sends the information to the results page](https://make.wordpress.org/hosting/test-results/); and the Reporter, which is the plugin that [works on the hosting page](https://make.wordpress.org/hosting/) and shows the results.
+
+## What is it
+
+### Runner
+
+Hosting companies can have hundreds of websites hosted with WordPress, so it's important to make sure their configuration is as compatible as possible with the software.
+
+To verify this compatibility, the WordPress Community provides a series of [PHPUnit](https://phpunit.de/) tests with which to check the operation of WordPress in any environment.
+
+### Reporter
+
+The Runner tests generates a report with the test results related to a bot user (a hosting company), and this captures and displays those test results at the [Host Test Result](https://make.wordpress.org/hosting/test-results/) page.
+
+## Try the PHPUnit Test Runner
+
+### What's the phpunit-test-runner
+
+The [phpunit-test-runner](https://github.com/WordPress/phpunit-test-runner) is a small test piece of PHPUnit specifically designed for hosting companies. There is a [whole documentation about this tool](https://make.wordpress.org/hosting/test-results-getting-started/). Also, if you want, you can make your test results appear in the [Host Test Results page](https://make.wordpress.org/hosting/test-results/) of WordPress.
+
+The tool can be run manually or through an automated system like Travis. To see how it works and the purpose of this document, will be shown how to run the tests manually.
+
+### Requirements
+
+To use the Runner, the following is required:
+* A server / hosting (infrastructure) with the usual configuration you have.
+* A database where you can test (it will be created and destroyed several times)
+
+### Installing the Runner
+
+First, download and synchronize the tool. The example use the `/wptestrunner/` folder, but use the best for you.
+
+```bash
+cd /wptestrunner/
```

Comment:
Making a folder at the root isn't probably going to work. Maybe something inside ~/, or /home/youruser/ ?

Reply:
Updated to "/home/wptestrunner/" and a coment: "This example use `/home/wptestrunner/` folder, but use the best for this machine."